seacrowd/sea_datasets/mlqa/mlqa.py

```diff
@@ -0,0 +1,223 @@
+import os
+import json
+from pathlib import Path
+from typing import Dict, List, Tuple
+import datasets
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses, DEFAULT_SOURCE_VIEW_NAME, DEFAULT_SEACROWD_VIEW_NAME
+
+_CITATION = r"""\
+@article{lewis2019mlqa,
+    author={Lewis, Patrick and O\{g}uz, Barlas and Rinott, Ruty and Riedel, Sebastian and Schwenk, Holger},
+    title={MLQA: Evaluating Cross-lingual Extractive Question Answering},
+    journal={arXiv preprint arXiv:1910.07475},
+    year={2019}
+}
+"""
+
+_DATASETNAME = "mlqa"
+
+_DESCRIPTION = """\
+MLQA (MultiLingual Question Answering) is a benchmark dataset for evaluating cross-lingual question answering performance.
+MLQA consists of over 5K extractive QA instances (12K in English) in SQuAD format in seven languages - English, Arabic, 
+German, Spanish, Hindi, Vietnamese and Simplified Chinese. MLQA is highly parallel, with QA instances parallel between 
+4 different languages on average.
+"""
+
+_HOMEPAGE = "https://github.com/facebookresearch/MLQA"
+_LICENSE = Licenses.CC_BY_SA_3_0.value 
+_URL = "https://dl.fbaipublicfiles.com/MLQA/"
+_DEV_TEST_URL = "MLQA_V1.zip"
+_TRANSLATE_TEST_URL = "mlqa-translate-test.tar.gz"
+_TRANSLATE_TRAIN_URL = "mlqa-translate-train.tar.gz"
+_SUPPORTED_TASKS = [Tasks.QUESTION_ANSWERING]  
+
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+class MLQADataset(datasets.GeneratorBasedBuilder):
+    """
+    MLQA (MultiLingual Question Answering) is a benchmark dataset for evaluating cross-lingual question answering performance.
+    MLQA consists of over 5K extractive QA instances (12K in English) in SQuAD format in seven languages - English, Arabic,
+    German, Spanish, Hindi, Vietnamese and Simplified Chinese. MLQA is highly parallel, with QA instances parallel between
+    4 different languages on average.
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    subsets = ["mlqa-translate-test.vi", 
+               "mlqa-translate-train.vi", 
+               "mlqa.vi.ar", 
+               "mlqa.vi.de", 
+               "mlqa.vi.zh", 
+               "mlqa.vi.en", 
+               "mlqa.vi.es", 
+               "mlqa.vi.hi", 
+               "mlqa.vi.vi",
+               "mlqa.ar.vi", 
+               "mlqa.de.vi", 
+               "mlqa.zh.vi", 
+               "mlqa.en.vi", 
+               "mlqa.es.vi", 
+               "mlqa.hi.vi",
+               "mlqa"]
```

Comment:
this `mlqa` subset is not handled in `_split_generators`,
maybe we should remove this?

Reply:
done